instance/tests/integration/integration_instance.py

```diff
@@ -244,6 +247,24 @@ def test_activity_csv(self):
         # stdout should contain 3 lines (as opposed to 2) to account for the last newline.
         self.assertEqual(len(out_lines), 3)
 
+    ''' FIXME JV : Need to add an additional test container?
+    @shard(4)
+    def test_activate_multiple_appservers(self):
+        """
+        Provision an instance, spawn and activate two AppServers
+        """
+        OpenEdXInstanceFactory(name='Integration - test_activate_multiple_appservers')
```

Comment:
We should be using `, use_ephemeral_databases=False` for all the integration tests other than the ephemeral test (the first one).